### PR TITLE
Update: key-spacing rule whitespace fixer (fixes: #6167)

### PR DESCRIPTION
**Pull request by RRMoelker**
See issue #6167

### CODE REVIEW

**Comment by RRMoelker** [comment]
I have signed the CLA with the same credentials as my commit. Does anybody know what else to try? Some way to persuade the CLA bot to check again?

**Comment by nzakas** [comment]
Lgtm. @btmills can you take a look?

File: lib/rules/key-spacing.js [hunk 1]
```diff
@@ -270,20 +272,59 @@ module.exports = {
...
                 !(expected && containsLineTerminator(whitespace))
             ) {
-                context.report(property[side], location, messages[side], {
-                    error: diff > 0 ? "Extra" : "Missing",
-                    computed: property.computed ? "computed " : "",
-                    key: getKey(property)
+                if (isExtra) {
+
+                    // Remove whitespace
+                    var range;
```
Comment (btmills) [inline lib/rules/key-spacing.js]: Can you move this up top with the other declarations?

**Comment by btmills** [comment]
@RRMoelker great job on this! Just one tiny change for consistency with the [code conventions](http://eslint.org/docs/developer-guide/code-conventions#variable-declarations), then LGTM as soon as that's done :ship:

File: lib/rules/key-spacing.js (hunk 1 above)
Comment (RRMoelker) [inline lib/rules/key-spacing.js]: Done

**Comment by btmills** [comment]
LGTM. Thanks @RRMoelker!

**Pull request merged**